PixelHunter1995/Main.cs

```diff
@@ -121,10 +122,9 @@ protected override void Update(GameTime gameTime)
 
             if (!musicPlaying)
             {
-                SoundEffectInstance soundInst = music.CreateInstance();
-                soundInst.Volume = 0.01f;
-                soundInst.IsLooped = true;
-                soundInst.Play();
+                MediaPlayer.IsRepeating = true;
```

Comment:
MediaPlayer is not guaranteed to exist, either include DLL in project, or change?